src/app/testing/gene-mocks.ts

```diff
@@ -3144,6 +3146,137 @@ export const geneMock2: Gene = {
     },
   ],
   nominations: 4,
+  is_adi: false,
+  is_tep: true
+};
+
+export const geneMock3: Gene = {
```

Comment:
From [the build log](https://app.travis-ci.com/github/Sage-Bionetworks/Agora/builds/265394612), it looks like this mock might be missing `nominatedtarget` and `nominations` fields:

```log
ERROR in src/app/testing/gene-mocks.ts:3153:14 - error TS2739: Type '{ _id: string; ensembl_gene_id: string; name: string; 
summary: string; hgnc_symbol: string; alias: string[]; isIGAP: false; haseqtl: true; isAnyRNAChangedInADBrain: true; 
rna_brain_change_studied: true; ... 5 more ...; is_tep: true; }' is missing the following properties from type 'Gene':
nominatedtarget, nominations

3153 export const geneMock3: Gene = {
                  ~~~~~~~~~
```

Reply:
Looks like I didn't commit all my changes.  Good catch!